web_timeline/readme/CONFIGURE.rst

```diff
@@ -26,6 +26,8 @@ the possible attributes for the tag:
 +--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 | colors             | No        | Allows to set certain specific colors if the expressed condition (JS syntax) is met.                                                                                                                                                                                      |
 +--------------------+-----------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| color-field        | No        | Allows to set a field (on the current model) that contains the HTML color (e.g. #FFFFFF). This option has priority over 'colors'.                                                                                                                                         |
```

Comment:
Perhaps changing this to `color_field` would be more appropriate since this is how it's used